shared/reducers/dev.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import {updateDebugConfig} from '../constants/dev'
+import type {State} from '../constants/reducer'
+import type {DebugConfig, DevAction} from '../constants/dev'
+
+type DevState = {
+  debugConfig: DebugConfig,
+}
+
+const initialState: DevState = {
+  debugConfig: {
+    dumbFilter: '',
+    dumbIndex: 0,
+    dumbFullscreen: false,
+  },
+}
+
+export default function (state: DevState = initialState, action: DevAction): State {
+  if (action.type === updateDebugConfig) {
```

Comment:
this is fine, just a note you could have used https://github.com/keybase/client/blob/9c6469ca7b17c8e18c1d7e2ed12f31ec46d6336c/shared/reducers/dev-edit.js#L42